components/admin-panel/sections/accounting/AccountingCategoryDrawer.tsx

```diff
@@ -108,6 +118,13 @@ function AccountingCategoryDrawerView(props: AccountingCategoryDrawerViewProps)
             <FormattedMessage id="AllExpenses" defaultMessage="All expenses" />
           )}
         </p>
+
+        <label className="mb-1 mt-4 text-base">
+          <FormattedMessage defaultMessage="Instructions" />
+        </label>
+        <pre>
+          <HTMLContent content={props.accountingCategory?.instructions} />
+        </pre>
```

Comment:
Removing this `pre` as it causes scrolling issues with long strings

![image](https://github.com/opencollective/opencollective-frontend/assets/1556356/1908c155-6a13-4854-b3f2-8f60d8108954)